lib/sda/host.go

```diff
@@ -387,14 +336,42 @@ func (h *Host) processMessages() {
 			m := data.Msg.(ServiceMessage)
 			h.processServiceMessage(data.Entity, &m)
 		default:
-			dbg.Error("Didn't recognize message", data.MsgType)
+			err := h.ProcessExternalMessage(&data)
```

Comment:
Oups - where does this comes from? I think this should be removed, now that services is running - no?